test/suites/nodeclaim/vm_extension_test.go

```diff
@@ -0,0 +1,38 @@
+package nodeclaim_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/samber/lo"
+
+	"sigs.k8s.io/karpenter/pkg/test"
+)
+
+var _ = Describe("VMExtension", func() {
+	It("should install all valid vm extensions", func() {
```

Comment:
Out of curiosity, do we still need to set the provisionMode to be bootStrappingClient/aksscriptless, or is this test for both?

Reply:
that isn't an option we are overriding here, this test will be executed on both contexts through the framework.